docs/bake-reference.md

```diff
@@ -481,6 +490,25 @@ FROM baseapp
 RUN echo "Hello world"
 ```
 
+### `target.description`
+
+Defines a human-readable description for the target, clarifying its purpose or
+functionality.
+
+```hcl
+target "lint" {
+    description = "Runs golangci-lint to detect style errors"
+    args = {
+        GOLANGCI_LINT_VERSION = null
+    }
+    dockerfile = "lint.Dockerfile"
+}
+```
+
+This attribute is useful when combined with the `targets` frontend method,
```

Comment:
This isn't actually `targets` frontend method, but `bake --list-targets` flag as these descriptions are for bake targets, not Dockerfile stages.

Reply:
Updated